internal/backup_fetch_handler.go

```diff
@@ -82,3 +100,13 @@ func HandleBackupFetch(folder storage.Folder,
 
 	fetcher(folder, backup)
 }
+
+func GetBackup(folder storage.Folder,
```

Comment:
1. Move this to `backup.go`?
2. Handle error at the top level (return an error instead of ...FatalOnError(err))
2. GetBackup name is too abstract, for example, you may name it like `SelectBackup/etc` to make it clear what is going on.